azkaban_cli/azkaban.py

```diff
@@ -715,3 +716,30 @@ def fetch_executions_of_a_flow(self, project, flow, start, length):
         self.__catch_response_error(response, FetchExecutionsOfAFlowError)
 
         return response.json()
+
+    def fetch_running_executions_of_a_flow(self, project, flow):
+        """Fetch running executions of a flow command, intended to make the request to Azkaban
+        and treat the response properly.
+
+        This method receives the project name and the flow id, making the fetch and evaluating the response.
+
+        Returns the json response from the request.
+
+        :param project: Project name on Azkaban
+        :type project: str
+        :param flow: Flow id on Azkaban
+        :type flow: str
+        :raises FetchRunningExecutionsOfAFlowError: when Azkaban api returns error in response
+        """
+
+        self.__check_if_logged()
+
+        response = api.fetch_running_executions_of_a_flow_request(
+            self.__session_id,
```

Comment:
Pass in `self.__session` and `self.__host`

Reply:
Done!